src/sentry/static/sentry/app/views/organizationMonitors/monitorModel.jsx

```diff
@@ -0,0 +1,18 @@
+import FormModel from 'app/views/settings/components/forms/model';
+
+export default class MonitorModel extends FormModel {
+    getTransformedData() {
+      return Object.entries(this.fields.toJSON()).reduce((data, [k, v]) => {
+        if (k.indexOf('config.') === 0) {
```

Comment:
Don't you need to set data for keys that don't start with `config`?

Reply:
oops - didnt test the non-config case clearly